README.md

```diff
@@ -36,6 +36,23 @@ $ gpg-group-chat --server
 $ gpg-group-chat --client
 ```
 
+## Basic usage flow
+
+1. Server starts with gpg-group-chat --server at 192.168.0.1
+2. The client1 starts with gpg-group-chat --client --server-ip=192.168.0.1 --public\_key=public\_key1
+3. The client1 connect to the server and the server stores the public key for client1
```

Comment:
s/The client1 connect to the server and the server stores the public key for client1/The client1 connect to the server and it stores the public key for client1